src/som/primitives/SystemPrims.java

```diff
@@ -216,6 +218,26 @@ public final Object doSObject(final String argument) {
     }
   }
 
+  @GenerateNodeFactory
+  @Primitive(primitive = "writeErrorMsg:")
```

Comment:
This is not very descriptive and undocumented.

What exactly is this supposed to do?
Does it mark the current message as having caused an error?

I am also not sure about terminology here.
Is this the message, or is it really about the turn?

Could be named `markTurnAsErroneous`?

Reply:
renamed to markTurnErroneous and added explanatory comment